sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -158,6 +164,12 @@ def test_regressor_parameter_checks():
                          " or an integer. 'invalid' was passed",
                          GradientBoostingRegressor(n_iter_no_change='invalid')
                          .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
+                         GradientBoostingRegressor(presort=5.324)
+                         .fit, X, y)
+    assert_raise_message(ValueError, "Presort must be a boolean",
```

Comment:
If there's no specific reason, please consider to remove this one. It seems duplicate.

Reply:
Should I remove the test for resort  = 5.324?

Reply:
My suggestion would be to have only one test (presort='invalid'), it seems unnecessary to have duplicate test for such minor thing :)

Reply:
Hmmm..seems legit.